client/src/api/api.js

```diff
@@ -248,12 +249,29 @@ export const uploadAudioFile = async (
 }
 
 /* Audio Endpoints */
+export const downloadAudioFileToExpo = async (vocabId, uri) => {
+  /* Download to a new URI based, with the filename including Date.now() in order to
+    React Native to rerender the audio. If the same filename is kept as before,
+    it won't change the audio displayed on the screen.
+  */
+  const splitPath = uri.split('.')
+  const fileType = splitPath.length > 2 ? splitPath[1] : 'caf'
+
+  const newURI = `${
+    FileSystem.documentDirectory
+  }${vocabId}-audio-${Date.now()}.${fileType}`
+  await FileSystem.copyAsync({ from: uri, to: newURI })
+  await putFileURI(vocabId, newURI, 'audio')
+
+  return { fileType }
+}
+
 export const downloadAudioFile = async (
```

Comment:
Why is the audio version called `downloadAudioFile` but the image version is called `uploadImageFile`? Wouldn't both of them be "uploading"?